src/metabase/lib/core.cljc

```diff
@@ -437,16 +441,23 @@
   clause?
   clause-of-type?
   fresh-uuids
+  mbql-stage?
   native-stage?
   normalized-query-type
+  normalized-mbql-version
   previous-stage
   previous-stage-number
   query-stage
   source-table-id
+  source-card-id
   update-query-stage]
  [metabase.lib.walk.util
   all-field-ids
   all-source-card-ids
   all-source-table-ids
+  all-template-tag-field-ids
+  all-template-tag-snippet-ids
   all-template-tags
+  all-template-tags-map
+  all-template-tags-id->field-ids
```

Comment:
All handful of utils for working with template tags (collection Field IDs, snippet IDs, etc)